### PR TITLE
Issue 81: update style for peer group email

### PR DESCRIPTION
**Pull request by kimschles**
## Migrations
NO

## Description
This pull request updates the styles of the peer group email 

## Github Issue
Fixes issue #81

## Background
This pull request has code for two different features: 
1) To update the styling of the peer_group_email
2) Adds directories and files to use Active Record to preview emails. I will add instructors for how to use preview in another pull request. 

## Screenshots
Before: http://snpy.in/PSKw8v
After: http://snpy.in/YBXd98

## Definition of Done

- [x] This PR has appropriate test coverage.
- [ ] Documentation has been updated. *(if needed)*

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def mentee_mail(mentor_match)
     mail(to: @mentee.email, subject: "You have requested to be mentored by #{@mentor.first_name}")
   end
 
-  def peer_mail(peer_group)
+  def peer_mail(peer_group)
```
Comment (rebeccafae) [inline app/mailers/user_mailer.rb]: extra space

File: app/models/mentorship.rb [hunk 2]
```diff
@@ -46,9 +46,14 @@ def send_mail
 
   def get_possible_mentors
     if mentee.stage_of_career == 5
-      User.where(mentor: true, waitlist: false, stage_of_career: 5).where( "mentor_times > ?", 0).where(mentor_industry: mentee.primary_industry).where("id != ?", mentee.id)
+      User.where(mentor: true, waitlist: false, stage_of_career: 5)
```
Comment (rebeccafae) [inline app/models/mentorship.rb]: 🥇

File: app/views/user_mailer/peer_mail.html.erb [hunk 3]
```diff
@@ -3,30 +3,47 @@
...
+    </h4>
+    <ul>
       <% @peers.each do |peer| %>
-        <li>
-          <b><%= peer.first_name %> <%= peer.last_name %></b>
-          <br><%= mail_to peer.email %> (Ask me about: <%= peer.top_3_interests.join(", ") %>)
-          <% if peer.linkedin_url %><br><%= link_to 'Connect on LinkedIn', peer.linkedin_url %><% end %>
-        </li>
+      <li>
+        <b><%= peer.first_name %>
```
Comment (rebeccafae) [inline app/views/user_mailer/peer_mail.html.erb]: can you use `strong` instead of `b`?

File: app/views/user_mailer/peer_mail.html.erb [hunk 4]
```diff
@@ -3,30 +3,47 @@
...
-		<p>The Women Rising Team</p>
+    <p>Happy solidarity and inspiration!</p>
+    <p>Love,</p>
+    <p>The Women Rising Team</p>
 
-		<p>Groups were formed based on your location. This formula is still in development, so feel free to reply with some <%= mail_to "info@womenrising.co", "Feedback" %>!<p>
-	</body>
-</div>
+    <p>PS Groups were formed based on your location. This formula is still in development, so feel free to reply with some
+      <%= mail_to "info@womenrising.co", "Feedback" %>!<p></body>
```
Comment (rebeccafae) [inline app/views/user_mailer/peer_mail.html.erb]: I think you meant `</p>`... looks like it was a mistake in the original code too

File: test/mailers/previews/user_mailer_preview.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class UserMailerPreview < ActionMailer::Preview
```
Comment (rebeccafae) [inline test/mailers/previews/user_mailer_preview.rb]: I'm not familiar with ActionMailer::Preview, how is this different from the letter_opener gem we already use?
Based on this article it seems we may want to stick with letter opener. https://dev.mikamai.com/2017/03/30/setting-an-environment-for-previewing-email-in-rails/

**Review by rebeccafae — APPROVED** [review]
There are a few minor nitpicks, but the email looks great! Thanks so much!

File: app/mailers/user_mailer.rb (hunk 1 above)
Comment (kimschles) [reply app/mailers/user_mailer.rb]: fixed.

File: app/views/user_mailer/peer_mail.html.erb (hunk 3 above)
Comment (kimschles) [reply app/views/user_mailer/peer_mail.html.erb]: 👍

File: app/views/user_mailer/peer_mail.html.erb (hunk 4 above)
Comment (kimschles) [reply app/views/user_mailer/peer_mail.html.erb]: fixed!

File: test/mailers/previews/user_mailer_preview.rb (hunk 5 above)
Comment (kimschles) [reply test/mailers/previews/user_mailer_preview.rb]: I didn't realize that the letter_opener gem was part of the project's dependencies. I'll get rid of ActionMailer.

File: Gemfile.lock [hunk 6]
```diff
@@ -1,10 +1,6 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionmailer (4.1.4)
```
Comment (rebeccafae) [inline Gemfile.lock]: We still need actionmailer

**Review by rebeccafae — CHANGES_REQUESTED** [review]

**Comment by rebeccafae** [comment]
@coloradokim please make sure actionmailer in still included, and then I will merge this

File: main.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+const STICKERS_URL = 'http://localhost:3000/api/v1/stickers'
```
Comment (rebeccafae) [inline main.js]: What are these stickers, I don't see that this route was created anywhere? Also, a localhost url will not work once on production

**Review by rebeccafae — CHANGES_REQUESTED** [review]
This looks good, other than the main.js file. I don't know if that was committed by mistake?

**Review by rebeccafae — APPROVED** [review]

**Pull request merged**